bin/run-browser-tests-local

```diff
@@ -4,4 +4,5 @@
 
 source bin/lib.sh
 
+cd browser-test/ || exit
```

Comment:
Thanks for fixing it!
Should you go back to the root folder after test completed?

Reply:
Changing directory only changes the directory for the scope of the script itself. The terminal calling the script isn't affected, so there's no need to cd back.